### DISCUSSION TITLE
Wordpress password list problem

### DISCUSSION THREAD

**Discussion by KillRamsey88** [community/community#64822] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys, how do I make wpscan take the txt file of wordpress passwords? It always gives me error, it says that the file does not exist. In which folder should I put it after downloading it from GitHub? Thank you

Using Ruby on Win11

**Comment by nirav-gajera** [comment]
To make wpscan take the txt file of WordPress passwords, you need to put the file in the same directory as the wpscan executable. You can download wpscan from the GitHub repository: https://github.com/wpscanteam/wpscan.

Once you have downloaded wpscan, extract the archive and you will find the executable in the `bin` directory.

To use wpscan with a txt file of WordPress passwords, you need to use the `-w` option. For example, to scan a website with the name `example.com` and a txt file of WordPress passwords named `passwords.txt`, you would use the following command:

```
wpscan -w passwords.txt example.com
```

If the txt file is not in the same directory as the wpscan executable, you need to specify the path to the file using the `-F` option. For example, to scan a website with the name `example.com` and a txt file of WordPress passwords named `/home/user/passwords.txt`, you would use the following command:

```
wpscan -w /home/user/passwords.txt example.com
```
